apps/ui/src/components/Onboarding.vue

```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+const usersStore = useUsersStore();
+const { web3 } = useWeb3();
+const followedSpacesStore = useFollowedSpacesStore();
+
+const user = computed(() => {
+  if (!web3.value.authLoading && web3.value.account) {
+    return usersStore.getUser(web3.value.account);
+  } else {
+    return null;
+  }
+});
+
+const tasks = computed(() => ({
+  profile: !user.value?.created,
+  following: followedSpacesStore.followedSpacesIds.length < 3,
+  votes: !user.value?.votesCount
+}));
+
+const hasPendingTasks = computed(() =>
+  Object.values(tasks.value).includes(true)
+);
+
+onMounted(async () => {
+  if (web3.value.account) await usersStore.fetchUser(web3.value.account, true);
```

Comment:
This `fetchUser` call is being force to be called on each page view.

This can be avoided by storing in localStorage a special value when all onboarding tasks are completed, and force the fetchUser only when this special value is not present.

Reply:
Nice idea, added this check